lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -822,6 +830,17 @@ def creds_search(*args)
       pass_regex = Regexp.compile(pass)
     end
 
+    if ptype
+      type = case ptype
+             when 'password'
+               Metasploit::Credential::Password
+             when 'hash'
+               Metasploit::Credential::PasswordHash
+             when 'ntlm'
+               Metasploit::Credential::NTLMHash
+             end
```

Comment:
rather than fall through here and just let ```type``` be nil and have the query silently run and not find anything, it might be more helpful to to let the user know what _would_ be accepted.  I'll add that when I land.